2010-09-25  Michael Tautschnig  <mt@debian.org>

	* setup-storage/Parser.pm: Clarify error message in case of invalid use of
		swap/luks (thanks Patrick Schoenfeld for pointing this out).
Index: trunk/lib/setup-storage/Parser.pm
===================================================================
--- trunk.orig/lib/setup-storage/Parser.pm
+++ trunk/lib/setup-storage/Parser.pm	
@@ -712,7 +712,8 @@
         mountpoint devices filesystem mount_options mdcreateopts
         | /^(luks|tmp|swap)\s+/
         {
-          ($FAI::device eq "CRYPT") or die "Encryted entry invalid in this context\n";
+          ($FAI::device eq "CRYPT") or
+            die "Encrypted device spec $1 invalid in context $FAI::device\n";
           defined ($FAI::configs{CRYPT}) or &FAI::internal_error("CRYPT entry missing");
 
           my $vol_id = 0;
